app/experimenter/experiments/constants.py

```diff
@@ -818,8 +818,10 @@ class ExperimentConstants(object):
 
 {experiment.short_description}
 
+Experimenter is the source of truth for details and delivery. Changes to Bugzilla are not reflected in Experimenter and will not change delivery configuration.
+
 More information: {experiment.experiment_url}
-        """
+        """  # noqa
```

Comment:
assuming the ``` # noqa``` allows flake8 and black to skip over it.
Thought it would be better to have the source of truth sentence as one long line since it looked funny being cut at ~79 characters in bugzilla?